.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: python
+python: 3.6
+install: pip install --upgrade --requirement requirements.txt
+script:
+  - rm --force --recursive api/checks && mkdir --parents api/checks && git clone --branch v3dev --depth 1 https://github.com/cs50/check50.git api/checks
+  - pushd api/checks; git filter-branch --subdirectory-filter docs/source && rm --force --recursive .git; popd
+  - make html && rm --force --recursive _build
+deploy:
+  provider: script
+  script: |
+    git remote set-url origin "https://$USERNAME:$PASSWORD@github.com/cs50/manual50.git" && \
```

Comment:
Public repo so shouldn't need credentials fyi

Reply:
We're pushing.